cosmos/x/evm/plugins/txpool/mempool/mempool_reader.go

```diff
@@ -122,6 +123,7 @@ func (etp *EthTxPool) queued() map[common.Address]coretypes.Transactions {
 //
 // NOT THREAD SAFE.
 func (etp *EthTxPool) Nonce(addr common.Address) uint64 {
+	exit := false
 	pendingNonces := make(map[common.Address]uint64)
```

Comment:
there is no need to store a map of addresses actually, we only care about the given `addr`

Reply:
nah Stats() returns everything

Reply:
what? this is just for the nonce function which only cares about the given `addr`

Reply:
oh lol ui glitched